source/Storyline.cpp

```diff
@@ -0,0 +1,179 @@
+/* Storyline.cpp
+Copyright (c) 2022 by Michael Zahniser
```

Comment:
you maybe want to add your name/username here

Reply:
Heh, happy to share the glory.